README.md

```diff
@@ -1,75 +1,169 @@
-ERT Storage Postgresql edition (tm) proof-of-concept
+ERT Storage Server
 ====================================================
 
-# Setup
-These instructions are for RHEL 7.
+This is the permanent storage solution for the
+[ERT](https://github.com/equinor/ert) project. It is written in Python 3.6+
+using FastAPI, Pydantic, SQLAlchemy, Graphene and Azure Blob Storage.
 
-## Setup PostgreSQL
-Install the required packages. On RHEL 7 these are for postgres 9, which is probably ok.
+# Simple Usage
+
+This section describes how to use ERT Storage for testing or development purposes. It uses an in-memory SQLite database with no Azure Blob Storage.
+
+First, install ERT Storage into a Python virtual environment:
 
 ``` sh
-sudo yum install postgresql-server postgresql-devel
+# Create the virtual environment (assuming you have Python 3.6+)
+python3 -m venv my-venv
+
+# Enable the environment
+source my-venv/bin/activate       # sh/bash/zsh
```

Comment:
All these look like series of steps instead as options only. Also when checking down the lines.

Reply:
Ye I figured. Do we even need to explain how to create a venv?

Also, what do you mean by "Also when checking down the lines"?

Reply:
"Also when checking down the lines" = 2 pip install lines should be options

Reply:
Maybe instead of "simple usage" as the header make it "Development environment"?